prep-css-layout-classes/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Layout Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+    /* stylelint-disable block-no-empty */
+    body {
+      font-family: sans-serif;
+    }
+
+    h1 {
+      text-align: center;
+    }
+
+    .container {
+      max-width: 992px;
+      margin: auto;
+    }
+
+    .row {
+      width: 100%;
+      display: flex;
+    }
+
+    .column {
+      display: flex;
+      flex-basis: calc(100% / 3);
+      padding-left: 8px;
+      padding-right: 8px;
+    }
+
+    img {
+      width: 100%;
+      max-height: 425px;
+      overflow: hidden;
+    }
+
+    div {
+      display: flex;
+      flex-wrap: wrap;
+      align-items: flex-start;
+    }
```

Comment:
Hi Brandon, it shouldn't be necessary to make all divs flex containers to achieve the desired layout. You should be able to remove this rule set.

Reply:
Hi Tim, I think I figured it out 👍 

![image](https://user-images.githubusercontent.com/109567499/183233334-f37b9a51-9ad3-41fc-a74a-c49fbb8552f9.png)